docs/src/modules/components/ScrollableTableOfContents.js

```diff
@@ -194,24 +122,12 @@ export default function AppTableOfContents(props) {
     if (active && activeState !== active.hash) {
       setActiveState(active.hash);
     }
-  }, [activeState]);
+  }, [activeState, items]);
 
   // Corresponds to 10 frames at 60 Hz
-  useThrottledOnScroll(itemsServer.length > 0 ? findActiveIndex : null, 166);
-
-  const handleClick = hash => event => {
-    // Ignore click for new tab/new window behavior
```

Comment:
I assume it comes from a rebase conflict. This logic is valuable. I think that we should keep it.